packages/pluggableWidgets/barcode-scanner-web/tests/e2e/specs/BarCodeScanner.spec.ts

```diff
@@ -0,0 +1,16 @@
+import page from "../../../../../../configs/e2e/src/pages/page";
+
+describe("barcode-scanner-web", () => {
+    beforeAll(() => {
+        page.open(); // resets page
+    });
+
+    it("compares with a screenshot baseline and checks if the media stream is started", () => {
+        browser.setWindowRect(0, 0, 1200, 900);
```

Comment:
I think we can prevent the media indicator doing `browser.setWindowRect(5, 5, 1195, 895);` then you can reduce the error to zero